README.md

```diff
@@ -10,91 +10,84 @@ Carnival is maintained and funded by [thoughtbot, inc](http://thoughtbot.com/com
 
 Thank you to all [the contributors](https://github.com/thoughtbot/carnival/contributors)!
 
-## Getting started
-
-Install ghc, alex, happy, cabal, Postgresql. On OS X, the easiest way is to
-install Haskell Platform using homebrew:
+## Pre-requisites
 
-    $ brew install haskell-platform
+Carnival is built locally, on CI, and deployed using [halcyon][]. Therefore,
+there is **no pre-requisite of a Haskell environment**, though you will need the
+following tools:
 
-Run the setup script first:
+- PostgreSQL
+- CoffeeScript
+- Heroku Toolbelt (needed for setup and deployment)
+- jq (needed for deployment)
+- `brew install bash coreutils` (if on OSX)
 
-    $ bin/setup
+[halcyon]: https://halcyon.sh/
 
-Then boot the site:
+## Getting started
 
-    $ yesod devel
+Carnival can be setup with one command:
 
-And run the tests:
+```
+$ ./bin/setup
+```
 
-    $ yesod test
+How long this takes depends on a number of things. All build artifacts are
+cached between anyone who builds Carnival (including Travis and Heroku). This
+means many builds can complete in under a minute. Unfortunately, the artifacts
+are specific to your platform. If you are the first person to build a given
+version of Carnival on your platform, it can take closer to 45 minutes.
 
-## Deployment
+## Developing
 
-Staging is the default deploy target:
+Before doing anything:
 
 ```
-$ ./bin/deploy
-```
-
-The production app can be passed as an argument if desired:
+$ source <(/app/halcyon/halcyon paths)
 
+# POSIX version
```

Comment:
Do you think it will be clear to everyone that they don't need to run both of these commands?